app/helpers/interview_path_helper.rb

```diff
@@ -0,0 +1,17 @@
+module InterviewPathHelper
+  def interview_path_for(application_choice, wizard, interview, step, params = {})
+    if step.to_sym == :referer
+      wizard.referer
+    elsif step.to_sym == :input
+      new_provider_interface_application_choice_interview_path(application_choice, params)
+    elsif step.to_sym == :edit
```

Comment:
when attempting to edit an existing interview, then going back, and then trying a setup a new interview, I'm getting an error here because the wizard's state has not been cleared. We should make sure to look out for this when moving to clearing the wizard tickets